src/pages/staticPages/tools/mutationMapper/StandaloneMutationTable.tsx

```diff
@@ -42,6 +42,8 @@ export default class StandaloneMutationTable extends MutationTable<
             MutationTableColumnType.CHROMOSOME,
             MutationTableColumnType.PROTEIN_CHANGE,
             MutationTableColumnType.MUTATION_TYPE,
+            MutationTableColumnType.CLONAL,
```

Comment:
Just curious, in the standalone mutation table, we do not include `CANCER_CELL_FRACTION`?

Reply:
We are considering dropping the changes in the `StandaloneMutationTable` and only support allele specific copy number visualization in the portal only. Whether we move forward with supporting ASCN visualization in the standalone tool depends on feedback from the frontend team. @onursumer @alisman @n1zea144 Can you comment?